webservice/tscanservice/tscanwrapper.py

```diff
@@ -273,28 +276,64 @@ def init_alpino_lookup(configfile, inputdir, outputdir):
     inputfiles.append(str(inputfile))
 
 
+def copy_modify_metadata(outputdir: str, source_filename: str, total_filename: str) -> None:
+    with open(os.path.join(outputdir, f".{source_filename}.METADATA"), 'r') as source_file:
+        with open(os.path.join(outputdir, f".{total_filename}.METADATA"), 'w') as target_file:
+            content = source_file.read()
+            content = content.replace("Document statistics", "Corpus statistics")
+            content = content.replace(re.sub(r'\.[^\.]+\.csv$', '', source_filename), "TOTAL")
+            target_file.write(content)
+
+
+def merge_output(outputdir: str, type: str) -> None:
+    first_file = True
+    total_filename = f"total.{type}.csv"
+
+    # create the empty target file
+    open(total_filename, 'w').close()
+
+    for f in glob.glob(f"{outputdir}/*.{type}.csv"):
+        filename = os.path.basename(f)
+        if filename == total_filename:
+            # ignore lingering total files
+            continue
+
+        if first_file:
+            # copy and modify metadata
+            copy_modify_metadata(outputdir, filename, total_filename)
+
+        first_line = True
+        with open(os.path.join(outputdir, total_filename), 'a') as target_file:
+            with open(os.path.join(outputdir, filename), 'r') as source_file:
+                while True:
+                    line = source_file.readline()
+                    if not line:
+                        break
```

Comment:
This will break on the first empty line (even if there are nonempty lines after it), is that intentional?